Changelog for v4.0.0-beta.4 (#5465)
apollographql/apollo-kotlin: CHANGELOG.md

Change Log
==========

# Version 4.0.0-beta.4

_2023-12-12_

## ✨ Initial Wasm support (#5458)

This release adds initial support for WebAssembly by adding the `wasmJs` target.

> Executing operations is working but this target is
> experimental ([Kotlin/Wasm](https://kotlinlang.org/docs/wasm-overview.html) is Alpha) and has some limitations:
> 
> - WebSockets are not implemented
> - No caching
> - No support for WASI or NodeJS
> - No tests

Reviewer: ```suggestion
Executing queries/mutations is working but this target is
experimental ([Kotlin/Wasm](https://kotlinlang.org/docs/wasm-overview.html) is Alpha) and has multiple limitations:

- No WebSockets
- No caching
- No support for WASI or NodeJS
```
